client/scripts/app.js

```diff
@@ -57,6 +66,13 @@ class Nametag extends Component {
       window.location = postAuth
       window.localStorage.removeItem('postAuth')
     }
+
+    // Attach user id to Sentry error reports
+    client.watchQuery({query: USER_QUERY}).subscribe({
```

Comment:
Ahh, looks like setting a fetchPolicy of `fetchOnly` will do the trick. I'll submit a commit.